src/ldp/representation/RepresentationMetadata.ts

```diff
@@ -35,4 +35,13 @@ export interface RepresentationMetadata {
    * Optional timestamp of the representation.
    */
   dateTime?: Date;
+  /**
+   * Optional link relationships of the representation.
+   */
+  linkRel?: { [id: string]: Set<string> };
```

Comment:
What is the difference between a missing object and an empty object?

Reply:
I'm afraid there shouldn't be any difference too, but I followed this comment https://github.com/solid/community-server/pull/52#discussion_r469044441
Do you want me to change it?